experimental/unitsconversion/tests/unitsTest.txt

```diff
@@ -0,0 +1,245 @@
+# Test data for unit conversions
```

Comment:
By convention, this should be located at `experimental/unitsconversion/tests/data/unitsTest.txt`

Reply:
done